tools/Custom/Module.cs

```diff
@@ -149,5 +170,10 @@ private async Task OnCmdletEndProcessing(string id, CancellationToken cancellati
                 }
             }
         }
+
+        private bool IsNetFramework()
```

Comment:
Is there a better way to detect that we are running in NETFramework?

Reply:
`RuntimeInformation.FrameworkDescription` should be enough to detect the name of the .NET installation an app is running on. See https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription?view=netframework-4.7.2.

I'll change the implementation to check the PS host version instead since we are already capturing it in `(Get-MgContext).PSHostVersion`.